distribution/lib/Standard/Base/0.0.0-dev/src/Data/Vector.enso

```diff
@@ -86,15 +87,8 @@ new_builder (capacity=1) = Builder.new capacity
 
    A vector allows to store an arbitrary number of elements in linear memory. It
    is the recommended data structure for most applications.
-
-   ! Value Copying
-     As Enso vectors implement immutable semantics, this constructor function
-     makes a copy of each value in the argument array.
-
-     If this didn't happen then it would be possible for the underlying array to
-     be mutated under the hood, and sneak mutability into our immutable data.
-from_array : Any -> Vector Any
-from_array arr = new (Polyglot.get_array_size arr) (arr.at _)
+from_polyglot_array : Any -> Vector Any
+from_polyglot_array arr = Vector (Proxy_Polyglot_Array.Proxy_Polyglot_Array arr)
```

Comment:
This would probably become simpler with the implementation of [Interop for Array & Vector](https://www.pivotaltracker.com/n/projects/2539304/stories/183000876).